bin/cassandra

```diff
@@ -188,6 +188,10 @@ launch_service()
         cassandra_parms="$cassandra_parms -Dcassandra-pidfile=$pidpath"
     fi
 
+    if [ "x$clean_heap_dump_files_defined" = "x1" ]; then
```

Comment:
@gowa why dont you just remove "x"? from both sides? I have never seen this construct.